compiler/plugins/target/AMD-AIE/iree-amd-aie/Target/CMakeLists.txt

```diff
@@ -24,6 +24,7 @@ iree_cc_library(
     iree::target::amd-aie::aie::AIEDialectIR
     iree::target::amd-aie::aie::AIEXDialectIR
     iree::target::amd-aie::aie::AIEPasses
+    iree::target::amd-aie::aie::AIEPassesFromMLIRAIE
```

Comment:
Can you remove this - `AIETargets` has `iree::target::amd-aie::aie::AIEPasses` which are the "in-tree" versions (which you already realized/know I'm sure).

Reply:
I need it for AIENormalizeAddressSpacesPass

Reply:
OH. Whoops. Sorry you're right. It's unfortunate that we've "crossed the streams here" but okay we'll live with it.

Reply:
I was able to move it and localize locks pass into the translation pipeline. So atleast a little bit uncrossed :)